### DISCUSSION TITLE
How to get fans to delay their response?

### DISCUSSION THREAD

**Discussion by Gil80** [Rem0o/FanControl.Releases#803] [Q&A]
I'm trying to configure the app to get the fans to respond to spikes in temps in a 5 seconds delay. Because the fans rev up and down for every small thing I do.
[userConfig.zip](https://github.com/Rem0o/FanControl.Releases/files/8730836/userConfig.zip)

**Comment by xhul-dev** [comment]
Hello there.
Most fan curve types have a "Response time" parameter available.
Increase the value to 5 seconds and see how it goes.
Cheers.

**Reply by Gil80** [reply]
Hi. 
Thank you. I've done that but still hear fan rev up quickly. Maybe it's the GPU?

**Reply by Rem0o** [reply]
^ Make sure to identify your fans, especially the one that rev-up, before trying to fix the issue with a configuration.
